.github/workflows/tests.yml

```diff
@@ -38,10 +35,11 @@ jobs:
     outputs:
       code: ${{ steps.filter.outputs.code }}
     steps:
-      - uses: actions/checkout@v3
-        # dorny/paths-filter needs git clone for push events
+      - uses: actions/checkout@v4
+        # dorny/paths-filter needs git clone for non-PR events
         # https://github.com/dorny/paths-filter#supported-workflows
         if: github.event_name != 'pull_request'
+
       - uses: dorny/paths-filter@v2.11.1
```

Comment:
The `filters` argument here needs to be updated to drop the `dev/` files.

Reply:
https://github.com/conda/conda/pull/13162/commits/e7fc559182b3c8dbd9271a664639e362f7c06ac3